wiki/People/Mapping_Assistance_Team/en.md

```diff
@@ -6,16 +6,27 @@ tags:
 
 # Mapping Assistance Team
 
-The **Mapping Assistance Team** (***MAT***) was a group responsible for assessing the quality of [pending](/wiki/Beatmap/Category#wip-and-pending) [beatmaps](/wiki/Beatmap) and recommending them for [ranking](/wiki/Beatmap/Category#ranked) in the [forum modding](/wiki/Modding/Forum_modding) system. Their recommendations were passed on to the [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*) by placing a [proto-bubble](/wiki/Modding/Proto-bubble), and the BAT would then make the final decision on whether the beatmap should enter Ranked.
+The **Mapping Assistance Team** (***MAT***) was a group responsible for assessing the quality of [pending](/wiki/Beatmap/Category#wip-and-pending) [beatmaps](/wiki/Beatmap) and recommending them for [ranking](/wiki/Beatmap/Category#ranked) in the [forum modding](/wiki/Modding/Forum_modding) system. Previously, mappers would go directly to a member of the [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*) for beatmap moderation, but now mappers would approach a MAT instead. MATs would serve as a buffer for quality assurance, before then passing on maps to the BAT. Their recommendations for beatmaps were passed to the BAT by placing a [proto-bubble](/wiki/Modding/Proto-bubble), and the BAT would then make the final decision on whether the beatmap should enter Ranked.
```

Comment:
```suggestion
The **Mapping Assistance Team** (***MAT***) was a group responsible for assessing the quality of [pending](/wiki/Beatmap/Category#wip-and-pending) [beatmaps](/wiki/Beatmap) and recommending them for [ranking](/wiki/Beatmap/Category#ranked) in the [forum modding](/wiki/Modding/Forum_modding) system. The MAT served as a first point of contact for mappers looking to rank their maps, where previously this would have been part of the role of the [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*). MATs passed their recommendations for ranking to the BAT by placing a [proto-bubble](/wiki/Modding/Proto-bubble) on the beatmap, and the BAT would then make the final decision on whether the beatmap should enter Ranked.
```
i want to avoid saying "now" to describe the updated situation, i get what you mean but it should still be clear that everything here is in the past. also reworded a bit to not duplicate as much from the first sentence in the paragraph